src/ontology/agro-edit.owl

```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/AGRO_00010164> "remnant trees"@en)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/AGRO_00010164> "Scattered trees may enhance the provision of ecosystem services that might benefit farmers and owners of rural properties, such as pollination of crops, shading for cattle, regulation of nitrogen dynamics and carbon sequestration, herbaceous production, and wood provision."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010164> "scattered trees"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010164> <http://purl.obolibrary.org/obo/AGRO_00000585>)
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010165> (taungya farming)
+
+AnnotationAssertion(Annotation(rdfs:comment "https://www.sciencedirect.com/science/article/abs/pii/S1462901111000451") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010165> "A forestry system that involves inter-planting trees with agricultural crops, particularly the local population’s staple foods, and so serves to satisfy the farmer’s quest for arable land."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010165> <https://orcid.org/0000-0003-3985-4911>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010165> "2022-01-04T00:56:20Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010165> "taungya farming"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010165> <http://purl.obolibrary.org/obo/AGRO_00002071>)
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010166> (forest garden)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "Crawford, Martin (2010). Creating a Forest Garden. Green Books. https://www.agroforestry.co.uk/wp-content/uploads/product/creating-a-forest-garden-2/Creating_a_forest_garden.pdf") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010166> "A garden structured on woodlands which have direct or indirect agricultural uses, and are self-maintained."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010166> <https://orcid.org/0000-0003-3985-4911>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010166> "2022-01-04T01:00:02Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> <http://purl.obolibrary.org/obo/AGRO_00010166> "Woodland garden"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010166> "forest garden"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010166> <http://purl.obolibrary.org/obo/AGRO_00000584>)
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010167> (shifting cultivation)
```

Comment:
Could you please import the PATO term : http://purl.obolibrary.org/obo/PATO_0000083 ## phase and create the term cropping phase of shifting cultivation, under this PATO term?

Reply:
Added issue for parent class 'Phase' import (#169)
Updated label and parent class to imported class. 
Commit e62bcd7e4088f6a460ab6e16ed8bb5e1ca08c75b